bindings/python/tests/fdbcli_tests.py

```diff
@@ -542,6 +542,121 @@ def triggerddteaminfolog(logger):
     output = run_fdbcli_command('triggerddteaminfolog')
     assert output == 'Triggered team info logging in data distribution.'
 
+@enable_logging()
+def tenants(logger):
+    output = run_fdbcli_command('listtenants')
+    print(output)
```

Comment:
Nit: You can use `logger.debug(output)`, `logger.info()` ...  to log the output and choose the logging level by your preference.

Reply:
Oh oops, I meant to remove these print statements